GameData/Kerbalism/Localization/fr-fr.cfg

```diff
@@ -0,0 +1,1146 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// NEVER EDIT THIS CFG FILE MANUALLY, IT IS AUTOGENERATED.
+// CHANGES MUST BE DONE IN THE CODE, INSIDE THE `Local` CLASS (LocalizationCache.cs)
+//////////////////////////////////////////////////////////////////////////////////////
+
+Localization
+{
+  en-us
```

Comment:
Change to fr-fr

Reply:
Thx for the update.
I don't mind working on a new translation later.